doc/radosgw/s3/bucketops.rst

```diff
@@ -862,7 +862,8 @@ Syntax
 Response Entities
 ~~~~~~~~~~~~~~~~~
 
-Response is XML encoded in the body of the request, in the following format:
+Response header contains ``LastModified`` date/time of the logging configuration..
+Logging configuration is XML encoded in the body of the response, in the following format:
```

Comment:
Suggest:

The response header ``LastModified`` provides the timestamp of most recent bucket logging configuration..
The configuration itself is XML encoded in the body of the response, in the following format: